src/main/java/org/folio/service/SynchronizationJobService.java

```diff
@@ -137,21 +156,29 @@ private Future<SynchronizationJob> rebuildUserSummaries(SynchronizationJob job)
       .filter(Objects::nonNull)
       .map(userSummaryService::rebuild)
       .collect(Collectors.toList()))
-      .map(job);
+      .map(job)
+      .onSuccess(result -> log.info("rebuildUserSummaries:: result: {}",
```

Comment:
this expression should fit into 100 chars, no?

Reply:
done